low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -25,17 +33,47 @@ class SSPLTestCmd:
     def __init__(self, args: list):
         self.args = args
         self.name = "sspl_test"
+        self.plan = "self_primary"
+        self.avoid_rmq = False
 
     def process(self):
-        # TODO: Need to convert run_tests.sh from shell to python
-        test_plan=None
-        sspl_test_plans = ["sanity", "alerts", "self_primary","self_secondary"]
-        for i in range(len(self.args)):
-            if self.args[i] in sspl_test_plans:
-                test_plan = self.args[i]
-        if test_plan is None:
-            test_plan="self_primary"
-        CMD = f"{TEST_DIR}/run_tests.sh test {test_plan}"
+        global_config = self.args.config[0]
+        self.plan = self.args.plan[0]
+        self.avoid_rmq = self.args.avoid_rmq
+
+        Conf.load("sspl", sspl_config_path)
+        Conf.load("sspl_test", sspl_test_config_path)
+
+        # Take back up of sspl test config
+        sspl_test_backup = '/etc/sspl_tests.conf.back'
+        shutil.copyfile(sspl_test_file_path, sspl_test_backup)
```

Comment:
why this copy? This is from legacy code, though am unable to see how its useful still?

Reply:
global_config will be copied to sspl_test in upcoming steps and we need to revert this change once sspl_test completed. So we are making copy of sspl_test as a backup.